components/kubearchive/production/stone-prod-p02/kustomization.yaml

```diff
@@ -116,11 +116,11 @@ patches:
                   - containerPort: 8081
                 resources:
                   limits:
-                    cpu: 500m
-                    memory: 256Mi
+                    cpu: 100m
```

Comment:
Is this downscale intentional?

Usually we don't enforce a limit on CPU, so a requiring component can take all the available left free. If not needed, feel free to remove it.

Reply:
I'm setting the limit because of the Guaranteed QoS, not that we ran into any eviction problem but still.